src/test/java/com/atlassian/jira/cloud/jenkins/buildinfo/pipeline/JiraSendBuildInfoStepTest.java

```diff
@@ -104,7 +106,6 @@ public void testStep() throws Exception {
         final WorkflowRun mockWorkflowRun = mockWorkflowRun();
         final TaskListener mockTaskListener = mockTaskListener();
         when(mockTaskListener.getLogger()).thenReturn(mock(PrintStream.class));
-        when(mockWorkflowRun.getAction(SCMRevisionAction.class)).thenReturn(scmRevisionAction());
```

Comment:
This wasn't required. so removed.

Reply:
I think it has changed when you refactored the IssueKeyExtractors